### PR TITLE
Feature show translation

### PR DESCRIPTION
**Pull request by kai-franke**
I would really appreciate feedback on my implementation of US3.

- The button component has no styling yet, because I want to wait for the design to be agreed upon.
- For the functionality of the "Show answer" button I didn't write a handleClick() function against the tasks from the user story, because I use the update function of the useState instead.

### CODE REVIEW

File: components/Card.js [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
+import { useState } from "react";
+import { Button } from "./Button";
 
 export default function Card({ voc }) {
+  const [showTranslation, setShowTranslation] = useState(false);
+
   return (
     <StyledSection>
-      <p>{voc.word}</p>
+      <p style={{ whiteSpace: "pre" }}>
```
Comment (Roland-Hufnagel) [inline components/Card.js:10]: why "pre"? I think it is better to wrap a longer translation to fit the card.

File: components/Card.js [hunk 2]
```diff
@@ -1,9 +1,27 @@
...
+        {voc.word}
+        {showTranslation && `\n = \n ${voc.translation}`}
+      </p>
```
Comment (Roland-Hufnagel) [inline components/Card.js:13]: I love your simple solutions ❤️

File: components/Button.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import styled from "styled-components";
+
+const Button = styled.button``;
+
+export { Button };
```
Comment (Roland-Hufnagel) [inline components/Button.js:5]: I think it doesn't make sense at the moment but we will need it later in the future.

**Review by Roland-Hufnagel — CHANGES_REQUESTED** [review]
Hi Kai. 
I really love your simple solutions. This is a very nice, small and simple PR. 💪
(Most people in my course have a passion to make things as complex as can be).
Nonetheless I left a few comment to think about...

File: components/Button.js (hunk 3 above)
Comment (kai-franke) [reply components/Button.js]: To make it make sense already now, I added some styling. 😁 
 But yes, this component will actually only become important in the future. I think that it should then also become a functional component.

File: components/Card.js (hunk 1 above)
Comment (kai-franke) [reply components/Card.js]: Good point! I need `white-space` to make the line break in the template string work in JSX. I have now changed it to `pre-wrap`, so the display of longer translations should also work, right?

**Review by Roland-Hufnagel — APPROVED** [review]
This looks very good now. We can talk about whiteSpace a bit in the next meeting 🙂

**Pull request merged**